Dockerfile

```diff
@@ -46,6 +46,7 @@ WORKDIR /var/www/monica
 # copy system files into their proper homes, and set file ownership
 # correctly
 ADD . .
+RUN rm -rf tests
```

Comment:
Could you move this line in the next `RUN` block? to limit the instruction number.

Reply:
I'm not sure to understand why. 

Don't we use https://github.com/jwilder/docker-squash ?

Reply:
Or another similar tool

Reply:
@Mokto nope! I was not aware of this tool. Anyway, for this dev image, it doesn't really matter